src/main/resources/application.conf

```diff
@@ -31,3 +31,33 @@ submission-monitor-dispatcher {
   # Set to 1 for as fair as possible.
   throughput = 1
 }
+
+// Basic defaults for kamon configuration. There are a plethora of things which you can set, see the Kamon docs
+// for more information. Note that at a minimum you'll need to fill in the "hostname" field below. Outside of hostname
+// you should make sure you know what you're doing before overriding any of the rest of these
+kamon {
+  metric.filters {
+    akka-dispatcher {
+      includes = [ "**" ]
+      excludes = [ ]
+    }
+
+    akka-actor {
+      includes = [ "rawls/**" ]
+      excludes = [ "*/system/**", "*/user/IO-**", "*kamon*" ]
+    }
+
+    akka-router {
+      includes = [ "**" ]
+      excludes = []
+    }
+  }
+
+  statsd {
+    // If you're using Kamon instrumentation you'll want need to specify your hostname (and potentially port) where
```

Comment:
Which in turn was largely copied from Agora.

If you leave it blank it defaults to 127.0.0.1

If you give it some bogus value it won't crash but it won't do anything useful either